apps/files_external/3rdparty/icewind/smb/src/System.php

```diff
@@ -62,7 +62,14 @@ protected function getBinaryPath(string $binary): ?string {
 			$result = null;
 			$output = [];
 			exec("which $binary 2>&1", $output, $result);
-			$this->paths[$binary] = $result === 0 && isset($output[0]) ? (string)$output[0] : null;
+
+			if ($result === 0 && isset($output[0])) {
+				$this->paths[$binary] = (string)$output[0];
+			} else if (is_executable("/usr/bin/$binary")) {
```

Comment:
```suggestion
			} elseif (is_executable("/usr/bin/$binary")) {
```
Not blocking but should still be fixed upstream

Reply:
@come-nc can you submit a PR upstream ?

Reply:
https://github.com/icewind1991/SMB/pull/122